cmd/oceantv/broadcast.go

```diff
@@ -307,14 +308,67 @@ func extStart(ctx context.Context, cfg *BroadcastConfig, log func(string, ...int
 
 // extStop uses the OffActions in the provided broadcast config to perform
 // external streaming hardware shutdown.
+// If an OffAction is a controller action, we don't perform it yet.
 func extStop(ctx context.Context, cfg *BroadcastConfig, log func(string, ...interface{})) error {
 	if cfg.OffActions == "" {
 		return nil
 	}
 
-	err := setActionVars(ctx, cfg.SKey, cfg.OffActions, settingsStore, log)
-	if err != nil {
-		return fmt.Errorf("could not set device variables to end stream: %w", err)
+	vars := strings.Split(cfg.OffActions, ",")
+	if len(vars) == 0 {
+		return errors.New("no off actions to perform")
+	}
+
+	for _, v := range vars {
+		// Get the off action MAC to see if it's for a controller, if so don't perform it yet.
```

Comment:
Small detail, but its checking if its for _THE_ controller registered to a broadcast, not just any controller.